Marsiple/Common/Extensions/UIBarButtonItem-Marsiple.swift

```diff
@@ -10,7 +10,10 @@ import UIKit
 
 extension UIBarButtonItem {
     static func createCommentItem(target: Any, action: Selector) -> UIBarButtonItem {
-        // TODO: - Localize
-        return UIBarButtonItem(title: "Comment", style: .plain, target: target, action: action)
+        return UIBarButtonItem(title: LocalizationKey.BarButtonItem.commentTitle.localized(), style: .plain, target: target, action: action)
+    }
+    
+    static func createSaveItem(target: Any, action: Selector) -> UIBarButtonItem {
```

Comment:
We could eject the `create` from the method name, as it's clear from the method signature what it does. 
What do you think?